packages/nyc-test-coverage/src/index.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  InjectedSinks,
+} from '@temporalio/worker';
+import { CoverageSinks } from './sinks';
+import libCoverage from 'istanbul-lib-coverage';
+
+export const coverageMap = libCoverage.createCoverageMap();
```

Comment:
After seeing that istanbul uses a global coverage object, I guess this isn’t too bad.
Why not merge into `global__coverage__` directly then?

Reply:
Because, weirdly enough, `global.__coverage__` is not a CoverageMap instance. I'm not familiar enough with nyc internals to explain exactly why. But in order to merge coverage in memory, we need to keep a separate CoverageMap instance to allow us to use `merge()`, and then replace `global.__coverage__` with a transformed version of the CoverageMap.

Reply:
In any case, I'd avoid yet another global